tests/test-recipes/split-packages/copying_files/meta.yaml

```diff
@@ -9,9 +9,12 @@ requirements:
 outputs:
   - name: my_script_subpackage
     files:
-      - subpackage_file1
-      - somedir
-      - "*.ext"
+      include:
+        - subpackage_file1
+        - somedir
+        - "*.ext"
+      exclude:
+        - "*3.ext"
```

Comment:
Should we replace the old test with a new one instead of adding a new case? Maybe I would add a new output in this meta.yaml and then test the different behaviors differently, if possible (e.g. how the list-of-str behaviour does copy _anything_ in PREFIX, but include/exclude doesn't?).

Reply:
I have updated the test recipe to include multiple outputs: one uses the old file matching and one uses the new file matching.